script.sh

```diff
@@ -72,11 +79,11 @@ cd $meteor_deploy_path
 # Running specified command
 if [ "${mode}" = "DEPLOY" ]; then
 	echo "Deploying using config from ${meteor_deploy_path}..."
-	mup deploy --verbose
+	mup deploy --verbose "$servers_option"
```

Comment:
Is this the correct syntax, or do you just want
```shell
mup deploy --verbose $servers_option
```

Reply:
updated